styles/_main.scss

```diff
@@ -0,0 +1,39 @@
+.main {
+  margin: 0;
+  padding: 0;
```

Comment:
To do a reset, it's best to do

```
* {
margin: 0,
padding: 0,
box-sizing: border-box
}
```

This way you can reset margins and paddings of all items in one pass

Reply:
Explain, please. What is reset?

Reply:
Every item comes with some value by default. This set of commands, resets every padding and margin of every item back to 0, so it gets easier for you to position things around the screen consistently. Also, `box-sizing: border-box` helps you with that too.

You can read more about it here: https://developer.mozilla.org/es/docs/Web/CSS/box-sizing